gemma-core/src/main/java/ubic/gemma/core/analysis/preprocess/SplitExperimentServiceImpl.java

```diff
@@ -338,7 +334,7 @@ public ExpressionExperimentSet split( ExpressionExperiment toSplit, Experimental
         ExpressionExperimentSet g = ExpressionExperimentSet.Factory.newInstance();
         g.setDescription( "Parts of " + toSplit.getShortName() + " that were split on " + splitOn.getName() );
         g.setName( toSplit.getShortName() + " splits" );
-        g.setTaxon( toSplit.getBioAssays().iterator().next().getSampleUsed().getSourceTaxon() );
+        g.setTaxon( toSplit.getTaxon() );
```

Comment:
This has to be reverted, if we split an experiment by taxon, we want to inherit the taxon from the bioassay so that each split will have the correct taxon.

Reply:
We should have a test case covering this.

Reply:
At the same time this is not about splitting by taxon and we do not use factor value for that purpose. I think it's fine to keep it as it is.

Reply:
On second thought, this does not belong here and should be proposed in a separate PR.